spec/out_amplitude_spec.rb

```diff
@@ -51,6 +54,9 @@
           event_type: tag,
           user_id: 42,
           device_id: 'e6153b00-85d8-11e6-b1bc-43192d1e493f',
+          price: 10.05,
+          quantity: 1,
+          revenue_type: 'sustainer',
```

Comment:
not sure if my question is relevant, i see there's a field for `revenue` above, but it's not here.